float32 v_norm
float32 a_norm
float32 t2go
---
bool ack
